server/config/rateLimit.js

```diff
@@ -0,0 +1,13 @@
+import rateLimit from 'express-rate-limit';
+
+const timeFrameInHours = 24;
+const maxNoOfRequests = 5000;
```

Comment:
Lets start with 500 requests for 1 hour. We can always update this as time goes.

```suggestion
const maxNoOfRequests = 500;
```